src/types.ts

```diff
@@ -19,22 +19,24 @@ export type Slice = ReturnType<typeof zarr.slice>;
 export type AxisSelection = number | Slice | null;
 export type FullSelection = AxisSelection[];
 export type UIntType = zarr.Uint8 | zarr.Uint16 | zarr.Uint32;
+export type IndexType = zarr.Uint32; // TODO: | zarr.Uint64;
```

Comment:
I moved the 64 bit TODO here but didn’t do it yet, since that means we have to deal with `bigint`s, which is out of scope for this one.